src/libecalc/presentation/yaml/yaml_types/components/yaml_pump.py

```diff
@@ -33,6 +55,10 @@ class Config:
 
     energy_usage_model: YamlTemporalModel[str]
 
+    # TODO: Currently we share the stream conditions between consumer system and pump. Might change if they deviate ...
+    # TODO: We may also want to enforce the names of the streams, and e.g. limit to 1 in -and output stream ...? At least we need to know what is in and out ...
+    stream_conditions: Optional[YamlConsumerStreamConditions]
```

Comment:
I still think we should fix the schema here, by not making stream_conditions optional.